spec/controllers/albums_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe AlbumsController, type: :controller do
+
+  let(:create_params) do {
+		album: {
+			name: "Harry Potter",
+			description: "Magic",
+      rank: 0
+		}
+	}
+  end
+	let(:album) { Album.create(create_params[:album]) }
+
+  describe "GET index" do
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET new" do
+    it "is successful" do
+      get :new
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET show" do
+    let(:album) do
+      Album.create(name: "Brawn")
+    end
```

Comment:
For something this short, I prefer the 1-line syntax that you use above: `let(:album) { Album.create(name: "Brawn") }`